wp-content/themes/core/assets/img/admin/blocks/layout-inline.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 134 92"><path d="M69.9 24.3V35h50.5V24.3H69.9zM95.2 31H73.4v-2.4h21.8V31zm17.2 1.3l2.4-4.2 2.4 4.2h-4.8zM69.9 56.9v10.7h50.5V56.9H69.9zm25.3 6.2H73.4v-2.4h21.8v2.4zm19.6 1.8l-2.4-4.2h4.9l-2.5 4.2zM69.9 68.9v10.7h50.5V68.9H69.9zm25.3 6.2H73.4v-2.4h21.8v2.4zm19.6 1.8l-2.4-4.2h4.9l-2.5 4.2zM119.3 36v18.7H70.9V36h48.4m1-1H69.9v20.7h50.4V35z"/><path d="M73.5 41.5H111v1.6H73.5zM73.5 38.4H111V40H73.5zM73.5 44.7h40.6v1.6H73.5zM73.5 47.8h37.3v1.6H73.5zM15.4 29.4h39v2h-39zM15.4 25.4h43v2h-43zM15.4 33.4h45v2h-45zM15.4 41.5h37.4v2H15.4zM15.4 37.5h43v2h-43zM15.4 53.5h37.5v2H15.4zM15.4 49.5h43v2h-43zM15.4 57.5h45v2h-45zM40.3 12.3h55.5v4.8H40.3z"/></svg>
```

Comment:
Curious if you're creating these SVGs, or where you're getting them from?

Reply:
They are the icons we used in panels. I'm just porting them over for now as placeholder.